brave/brave-core
# Android using Desktop VPN credentials

Fixes https://github.com/brave/brave-browser/issues/20374

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Purchase VPN on Desktop via https://account.bravesoftware.com
2. Switch to an Android device for the rest of the test plan
3. If VPN doesn't show in hamburger menu use the QA menu to enable VPN
4. Navigate to accounts website (ex: https://account.bravesoftware.com/)
5. After logging in, you should see the VPN product showing
6. Purchased VPN product (from step 1) will show a button `Refresh Brave VPN`. Click this
7. Now try to enable VPN (it should show up on hamburger menu)
8. You should be prompted to add a profile to your device - and VPN should connect
9. Verify you're connected by visiting https://whatismyipaddress.com/
10. Try changing servers (in settings) and verify via https://whatismyipaddress.com/
